app/src/main/java/nodomain/freeyourgadget/gadgetbridge/externalevents/NotificationListener.java

```diff
@@ -1,6 +1,6 @@
 /*  Copyright (C) 2015-2018 Andreas Shimokawa, Carsten Pfeiffer, Daniele
     Gobbetti, Frank Slezak, Hasan Ammar, Julien Pivotto, Kevin Richter, Normano64,
-    Steffen Liebergeld, TaaviE, Zhong Jianxin
+    Steffen Liebergeld, Taavi Eomäe, Zhong Jianxin
```

Comment:
This would be overwritten by the next run of the License bot script. I fixed the mailmap but please ping me explicitly the next time to avoid misunderstandings!